testfiles/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.17
+
+# Relative path conversion top directories.
+set(CMAKE_RELATIVE_PATH_TOP_SOURCE "/home/jermy/CLionProjects/EnergyPlus")
+set(CMAKE_RELATIVE_PATH_TOP_BINARY "/home/jermy/CLionProjects/EnergyPlus")
+
+# Force unix paths in dependencies.
+set(CMAKE_FORCE_UNIX_PATHS 1)
+
+
+# The C and CXX include file regular expressions for this directory.
+set(CMAKE_C_INCLUDE_REGEX_SCAN "^.*$")
+set(CMAKE_C_INCLUDE_REGEX_COMPLAIN "^$")
+set(CMAKE_CXX_INCLUDE_REGEX_SCAN ${CMAKE_C_INCLUDE_REGEX_SCAN})
+set(CMAKE_CXX_INCLUDE_REGEX_COMPLAIN ${CMAKE_C_INCLUDE_REGEX_COMPLAIN})
```

Comment:
Hmmm, something went awry here @jmythms .  Looks like you accidentally pointed a build folder to the testfiles folder at some point and CMake created these files.  The files that need to be plucked back off this branch include:
 - testfiles/CMakeFiles/CMakeDirectoryInformation.cmake
 - testfiles/CMakeFiles/progress.marks
 - testfiles/CTestTestfile.cmake
 - testfiles/Makefile
 - testfiles/cmake_install.cmake